packages/cppo/cppo.1.6.0/opam

```diff
@@ -15,6 +15,7 @@ build-test: [
 ]
 
 depends: [
+  "ocaml"    {< "4.07"}
```

Comment:
For opam version 1.2 files, the way to mark this is in a separate `available` list.  For example, see [the available stanza in this 0install opam](https://github.com/ocaml/opam-repository/blob/master/packages/0install/0install.2.9.1/opam#L27).

Reply:
Thanks. That makes it easier to update a bunch of opam files.